File: app/src/main/java/com/github/weatherapp/core/Weather.java
package com.github.weatherapp.core;

public class Weather {
    public String main;

    public String icon;

    public String description;

    @Override
    public String toString() {
        return "Weather{" +
                "weatherData='" + main + '\'' +
                ", icon='" + icon + '\'' +
                ", description='" + description + '\'' +
                '}';
    }
}

File: app-kotlin/src/main/kotlin/com/github/weatherapp/core/Weather.kt
package com.github.weatherapp.core

class Weather {
    var main: String? = null

    var icon: String? = null

    var description: String? = null

    override fun toString(): String {
        return "Weather{" +
                "weatherData='" + main + '\'' +
                ", icon='" + icon + '\'' +
                ", description='" + description + '\'' +
                '}'
    }
}
